test/podFailover/README.md

```diff
@@ -17,50 +17,31 @@ export REGISTRY=<your-personal-docker-repo>
 make pod-failover-test-containers
 ```
 
-### Create metrics service
-
-- Create the metrics service where the workload pod can publish it's metrics using the command mentioned below:
-
-```console
-wget https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/test/podFailover/metrics-svc.yaml
-```
-
-- Edit the value of metrics-service-image and run:
-
-```console
-kubectl apply -f metrics-svc.yaml
-```
-
-- Create the service monitor to enable Prometheus to scrape the pod failover service metrics.
+### Run failover test
 
-```console
-wget https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/test/podFailover/service-monitor.yaml
-kubectl apply -f service-monitor.yaml
+- Deployment via helm chart (controller/metrics)
+``` console
+helm install pod-failover-controller pod-failover-controller-chart/ --values pod-failover-controller-chart/values.yaml
 ```
 
-### Run failover test
+- Values/Parameters - controllerPodImage(required), metricPodImage(optional, used for prometheus metrics)
 
-- Get the deployment file to deploy the controller
+- Deployment via helm chart (workload)
 
 ```console
-wget https://raw.githubusercontent.com/abhisheksinghbaghel/azuredisk-csi-driver/main_v2/test/podFailover/deployment.yaml
+helm install pod-failover-workload pod-failover-workload-chart/ --values pod-failover-workload-chart/1pod1pvc.yaml --set azureClientId=$AZURE_CLIENT_ID --set azureClientSecret=$AZURE_CLIENT_SECRET --set azureTenantId=$AZURE_TENANT_ID
```

Comment:
Do we really need a separate Helm chart for the workload, or can we just combine it with the controller chart?

Reply:
I originally had them put together but since helm charts can only generate resources once. When I wanted to run different workloads, the next workload would fail because it said the controller already exists

Reply:
Makes sense.